json/stepperMotor.json

```diff
@@ -1,52 +1,57 @@
 {
+  "meta": {
+    "scriptGit": "https://gist.github.com/cb9e4207e37668c776eaae643701f92c.git",
+    "actuator": "true",
+    "scriptFile": "nemaStepper.groovy"
+  },
   "GenericNEMA11": {
-    "flangeDiameter": 21.0,
+    "mountScrew": "M3",
     "flangeHeight": 2.0,
-    "length": 27.0,
```

Comment:
There is one potential benefit of CSV -> JSON. This is how it is sorted currently: https://github.com/sjkelly/Hardware-Dimensions/blob/901e65dac42357b5c0178d6347abbec9775cfca4/csv_to_json.py#L28